ntbs-service/Services/AzureAdDirectoryService.cs

```diff
@@ -260,13 +261,17 @@ public async virtual Task<IEnumerable<Microsoft.Graph.User>> GetUsersInGroup(str
             if(IsUserExternal(graphUser)){
                 userName = !String.IsNullOrEmpty(graphUser.Mail) ? graphUser.Mail : graphUser.UserPrincipalName;
             }
-            
+
+            var displayName = !String.IsNullOrEmpty(graphUser.DisplayName)
+                ? NameFormattingHelper.FormatDisplayName(graphUser.DisplayName)
+                : $"{graphUser.GivenName} {graphUser.Surname}";
```

Comment:
There is logic here that it would be good to add a test for

Reply:
Yeah I agree it would be nice, but I'm not sure where I would put it. I can't find any tests at all for this `AzureAdDirectoryService` class, and it could be quite tricky to start from scratch given my understanding of the class is still a bit shaky. What do you think I should do?

Reply:
Hmm, that's a tricky one. If testing this class is tricky, the other thing you could do is move the logic into the `NameFormattingHelper` (i.e. have a method there which takes `graphUser` as the parameter). However, that feels inappropriate as it doesn't feel like a concern the helper should know about. So maybe we have to agree that it's not worth it.